src/parser.rs

```diff
@@ -378,14 +378,15 @@ impl<'parser> Parser<'parser> {
                 } else {
                     // All block comment lines are indented at least beyond the "/*", so strip the
                     // indent and re-indent when formatting.
+                    let line_count = content.lines().count();
                     let trimmed_lines = content
                         .lines()
                         .enumerate()
                         .map(|(index, line)| {
                             if index == 0 {
                                 line
-                            } else if line.trim().len() == 0 {
-                                ""
+                            } else if index == line_count - 1 && line.trim() == "" {
+                                line.trim()
```

Comment:
I think these lines are also from the recent PR. Rebasing from upstream should fix both of these.

There were also changes to at least one test. It explains why tests don't pass. I think if you rebase from upstream the tests will probably pass.